cclib/parser/nwchemparser.py

```diff
@@ -1261,6 +1282,27 @@ def extract(self, inputfile, line):
                 self.append_attribute("vibirs", float(line.split()[5]))
                 line = next(inputfile)  # next line
             self.set_attribute("vibfreqs", vibfreqs)
+        # properties related to symmetry
+        if line.strip().find('symmetry #') != -1:

```

Comment:
I don't think this matches anything.  Do you know what this block is and what "symmetry number" is?  Number of irreps in the PG?

Reply:
Hm this is also present in `cclib/data/NWChem/basicNWChem7.0/dvb_ir.out`. It is on line 1997, though it's not clear what it represents yet.

Reply:
Okay, yes it is symmetry factor that goes into calculating the rotational entropy. it is related to the point group, but determined as discussed on [this page](http://molecularmodelingbasics.blogspot.com/2012/12/conformational-and-rotational-entropy.html).

I think this should be removed from this pull request and instead added in the follow when we revisit #1143 and discuss on how to organize these new pieces of information.